src/main/java/seedu/address/logic/commands/SplitCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Splits an amount of money with an existing person in the address book.
+ */
+
+public class SplitCommand extends Command {
+
+    public static final String COMMAND_WORD = "split";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Splits the amount of money " + "with a person. "
```

Comment:
Is there a missing element between ```"... money " + "with a person. "```? Also, might want to consider adding ```\n``` at end of this string